api/base/exceptions.py

```diff
@@ -80,20 +80,83 @@ class Conflict(APIException):
     default_detail = ('Resource identifier does not match server endpoint.')
 
 
-class InvalidQueryStringError(JSONAPIException):
+class JSONAPIParameterException(JSONAPIException):
+    def __init__(self, detail=None, parameter=None):
+        source = {
+            'parameter': parameter
+        }
+        super(JSONAPIParameterException, self).__init__(detail=detail, source=source)
+
+
+class JSONAPIAttributeException(JSONAPIException):
+    def __init__(self, detail=None, attribute=None):
+        source = {
+            'pointer': '/data/attributes/{}'.format(attribute)
+        }
+        super(JSONAPIAttributeException, self).__init__(detail=detail, source=source)
+
+
+class InvalidQueryStringError(JSONAPIParameterException):
     """Raised when client passes an invalid value to a query string parameter."""
     default_detail = 'Query string contains an invalid value.'
     status_code = http.BAD_REQUEST
 
-    def __init__(self, detail=None, parameter=None):
-        super(InvalidQueryStringError, self).__init__(detail=detail, source={
-            'parameter': parameter
-        })
 
+class InvalidFilterOperator(JSONAPIParameterException):
+    """Raised when client passes an invalid operator to a query param filter."""
+    default_detail = 'Invalid filter operator, must use one of: >, >=, =, <=, <.'
```

Comment:
The error message should use the actual letter inputs that are valid, e.g. `eq`, etc.